flamedisx/xenon/x1t_sr1.py

```diff
@@ -328,6 +365,13 @@ def add_extra_columns(self, d):
                           d['y_fdc'].values,
                           d['z_fdc'].values]))
 
+        if self.variable_se_gain:
+            d['se_gain_relative']=self.se_gain_map(np.transpose([d['x'].values,
```

Comment:
So the SE gain maps are meant to be read from the MC truth positions?

Reply:
Yes, for events at given (x,y) we get a value that allow to scale the default se_gain depending on the position.

Reply:
I think we can resolve this review since we found the wiki note and corrected the implementation in fd. 
Cheers :smile:

Reply:
Thanks @eangelino for updating this one line https://github.com/FlamTeam/flamedisx/blob/d47218c73fb6d4dbb3c28d183a841b60fa7cc5d4/flamedisx/xenon/x1t_sr1.py#L369